next.config.mjs

```diff
@@ -68,15 +77,10 @@ const nextConfig = {
       layers: true,
     };
 
-    return config;
-  },
+    config.resolve.alias['@ant-design/cssinjs'] = '@ant-design/cssinjs/lib';
```

Comment:
It's an awsome idea! I have tried with `transpilePackages` but find it not working correctly. 😅

cc @Madccc

Reply:
666